deploy/stacks/pipeline.py

```diff
@@ -941,15 +956,18 @@ def set_albfront_stage(self, target_env, repository_name):
         )
 
     def evaluate_post_albfront_stage(self, target_env):
-        if target_env.get("enable_cw_rum", False):
-            post=[
-                self.cognito_config_action(target_env),
-                self.cw_rum_config_action(target_env),
-            ]
-        else:
-            post=[
-                self.cognito_config_action(target_env),
-            ]
+        post = []
+        if target_env.get('custom_auth') is None:
+            post.append(self.cognito_config_action(target_env))
+
+        if target_env.get('enable_cw_rum', False):
+            post.append(self.cw_rum_config_action(target_env))
+
+        if len(post) == 0:
```

Comment:
Yes, this codeBuild step is not doing anything important , I have added this because the `post` step in code pipeline expected a Codebuildstep. let me see if I can refactor the code to remove this

Reply:
Refactored this so that we avoid the `len(post)` check.